File: c#/src/Docmost.Sdk/Models/PageInfoDto.cs
namespace Docmost.Sdk.Models;

public class PageInfoDto
{
    [System.Text.Json.Serialization.JsonPropertyName("pageId")]
    public string PageId { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("includeSpace")]
    public bool? IncludeSpace { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("includeContent")]
    public bool? IncludeContent { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("format")]
    public string Format { get; set; }

}

File: go/docmost/models/page_info_dto.go
package models

// PageInfoDto from OpenAPI.
type PageInfoDto struct {
	PageId string `json:"pageId"`
	IncludeSpace *bool `json:"includeSpace,omitempty"`
	IncludeContent *bool `json:"includeContent,omitempty"`
	Format *string `json:"format,omitempty"`
}
